templates/taxi/driver_list.html

```diff
@@ -4,7 +4,7 @@
   <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
-      {% for driver in driver_list%}
+      {% for driver in driver_list %}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The quotation marks around the URL name in the `{% url %}` tag should be single quotes to maintain consistency with Django's template syntax. Change `"taxi:driver-detail"` to `'taxi:driver-detail'`.